src/coin-selection/test/Spec.hs

```diff
@@ -614,3 +617,50 @@ stakeKeyWithdrawalTest = do
 
   -- withdraw rewards
   void $ tryBalanceAndSubmit mempty Wallet.w2 withdrawalTx TrailingChange [C.WitnessStakeKey stakeKey]
+
+addScriptWithdrawalTest
+  :: forall era m
+   . ( MonadIO m
+     , MonadMockchain era m
+     , MonadError (BalanceTxError era) m
+     , MonadFail m
+     , C.IsConwayBasedEra era
+     , C.HasScriptLanguageInEra C.PlutusScriptV2 era
+     )
+  => m ()
+addScriptWithdrawalTest = C.conwayEraOnwardsConstraints @era C.conwayBasedEra $ do
+  void registerScriptStakingCredential
+
+  let script = Scripts.v2StakingScript
```

Comment:
Nit: We could just inline `Scripts.v2StakingScript` here esp. as the variable isn't used in the next line